config/legacy/parameters.yaml.dist

```diff
@@ -43,10 +43,12 @@ parameters:
     # - registration - user "ra"
     # - readonly access to all endpoints - user "apireader"
     # - management - user "management"
+    # - deprovision - user "deprovision"
```

Comment:
Will the same user be used for the userinformation endpoint? In that case, we should choose another name

Reply:
The user is now named: lifecycle, the parameter to set its password is updated to: `lifecycle_password`